core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -776,6 +777,9 @@ public void run() {
                 if (!isCancelled()) {
                     logger.warn("Current thread for topic-partition-id {} is interrupted. Reason: {}", topicIdPartition, ex.getMessage());
                 }
+            } catch (RetriableException ex) {
+                logger.debug("Encountered a retryable error while executing current task for topic-partition {}. " +
+                    "Reason: {}", topicIdPartition, ex.getMessage());
```

Comment:
Let us print the complete exception, which prints stack trace too. That will be helpful while debugging.